source/includes/compatibility/language-compatibility-table-c.rst

```diff
@@ -0,0 +1,41 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility
+   :widths: 40 15 15 15 15
+
+   * - C Driver Version
+     - C17/C18
+     - C11
+     - C99
+     - C89
+   * - 1.28
+     - ✓
+     - ✓
+     - ✓
+     - 
+   * - 1.27
+     - ✓
+     - ✓
+     - ✓
+     - 
+   * - 1.26
+     - ✓
+     - ✓
+     - ✓
+     - 
+   * - 1.25
+     - ✓
+     - ✓
+     - ✓
+     -
+   * - 1.24
+     - ✓
+     - ✓
+     - ✓
+     -
```

Comment:
S: consolidate this version with all above versions

```suggestion
   * - 1.24 to 1.28
     - ✓
     - ✓
     - ✓
     -
```

Reply:
Do you think it's also worth shortening the MDB table to follow this pattern as well? I did notice several versions can be grouped together.